.github/workflows/workflow-get-latest-deployed-versions-from-github.yml

```
@@ -0,0 +1,51 @@
+    outputs:
+      infra_version_sha: ${{ steps.get-infra-sha.outputs.sha }}
+      apps_version_sha: ${{ steps.get-apps-sha.outputs.sha }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Get Latest Deployed Infra Version
+        id: get-infra-version
+        run: echo "version=$(gh variable get INFRA_VERSION_${{ inputs.environment }} --repo ${{ github.repository }} --json value --jq .value || echo '')" >> $GITHUB_OUTPUT
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:146: Double quote to prevent globbing and word splitting

(shellcheck)